documentation/modules/security/proc-replacing-your-own-private-keys.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-replacing-your-own-private-keys-{context}']
+= Replacing private keys used by your own CA certificates
+
+This procedure describes how to renew CA certificates and keys you installed yourself, instead of using the certificates generated by the Cluster Operator.
+Follow this procedure when you are changing the private keys used to sign the new CA certificates.
+
+If you are using your own certificates, the Cluster Operator will not renew them automatically.
+Therefore, it is important that you follow this procedure during the renewal period of the certificate in order to replace CA certificates that will soon expire.
+
+The procedure describes the renewal of CA certificates in PEM format.
+
+.Prerequisites
+
+* The Cluster Operator is running.
+* xref:installing-your-own-ca-certificates-{context}[Your own CA certificates and private keys are installed].
+* You have new cluster and clients X.509 certificates and keys in PEM format.
+
+.Procedure
+
+Before going through the following steps, make sure that the CN (Common Name) of the new CA certificate is different from the current one.
+For example, when the Cluster Operator renews certificates automatically it adds a _v<version_number>_ suffix to identify a version.
+Do the same with your own CA certificate by adding a different suffix on each renewal.
+By using a different key to generate a new CA certificate, retain the current CA certificate stored in the `Secret`.
+
+. Update the corresponding `Secret` retaining the current CA certificate, adding the new CA certificate and updating the generation.
++
+.. Edit the existing `Secret`:
++
+[source,shell,subs="+quotes"]
+kubectl edit secret _CA-CERTIFICATE-SECRET_
++
+_CA-CERTIFICATE-SECRET_ is the name of the `Secret`, which is `_KAFKA-CLUSTER-NAME_-cluster-ca-cert` for the cluster CA certificate and `_KAFKA-CLUSTER-NAME_-clients-ca-cert` for the clients CA certificate.
++
+Following an example of the `Secret` related to the cluster CA certificate for a Kafka cluster named `my-cluster`.
++
+[source,yaml,subs=attributes+]
+----
+apiVersion: v1
+kind: Secret
+data:
+  ca.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0F... <1>
+metadata:
+  annotations:
+    strimzi.io/ca-cert-generation: "0" <2>
+  labels:
+    strimzi.io/cluster: my-cluster
+    strimzi.io/kind: Kafka
+  name: my-cluster-cluster-ca-cert
+  #...
+type: Opaque
+----
+<1> Current CA certificate base64 encoded
+<2> Current CA certificate generation
```

Comment:
Sema as in the other procedure => showing both and after secret would be useful. Especially since the modification here goes beyond just changing the generation.